website/index.html

```diff
@@ -33,31 +33,38 @@ <h2>Test cases</h2>
       <div>
           <h2>Output</h2>
           <div id="output" class="mono-box">
-              <p id="stderr"></p>
-              <p id="stdout"></p>
+            <pre>
+              <div id="stderr">Error: Testing</div>
+              <div id="stdout">Hello, world!</div>
+            </pre>
```

Comment:
Wrapping this whole thing in `<pre>` causes some unwanted newlines and whitespace, changing the inner `div`s to `pre` is probably what we want instead?

Reply:
Whoops, that was supposed to be a div with `<pre>`s inside